src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.ts

```diff
@@ -82,7 +82,7 @@ export class CephfsClientsComponent implements OnInit {
         this.modalRef.hide();
         this.notificationService.show(
           NotificationType.success,
-          this.i18n('Evicted client "{{clientId}}"', { clientId: clientId })
+          this.i18n("Evicted client '{{clientId}}'", { clientId: clientId })
```

Comment:
It seems linters will allow `double quotes` when `single quotes` are used inside.
I would prefer to use backticks here, as this will reduce future work when we are able to migrate from `i18n` to `$localize` methods.

```suggestion
          this.i18n(`Evicted client '{{clientId}}'`, { clientId: clientId })
```

Reply:
@tspmelo should we replace all instances of `double quotes` with `backticks` or just the notification?

Reply:
I would suggest just the ones you are already changing.

Reply:
I just investigated a bit more and it looks like we can enforce this by changing the current `tslint` rule and we don't have that many cases as I though.
In `tslint.json` change the `quotemark` rule to `"quotemark": [true, "single"]`